### ISSUE TITLE
Update for tokenized API?

### ISSUE THREAD

**Issue by staze** [mike-levenick/stupid-groups#1]
Just curious if this tool will be updated to handle tokenized API access. 

Thanks!

**Comment by sbryant82** [comment]
Yes, at this time, this should be done via an API client ID and Client Secret.
